[2022-08-23 11:47:32,095] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-19T11:00:00+00:00 [queued]>
[2022-08-23 11:47:32,113] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-19T11:00:00+00:00 [queued]>
[2022-08-23 11:47:32,114] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:47:32,115] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:47:32,115] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:47:32,140] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-19 11:00:00+00:00
[2022-08-23 11:47:32,147] {standard_task_runner.py:52} INFO - Started process 853 to run task
[2022-08-23 11:47:32,152] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-19T11:00:00+00:00', '--job-id', '2298', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmpjk4tj886', '--error-file', '/tmp/tmpa2oi4kcl']
[2022-08-23 11:47:32,153] {standard_task_runner.py:80} INFO - Job 2298: Subtask print_world
[2022-08-23 11:47:32,263] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-19T11:00:00+00:00 [running]> on host fc645605a611
[2022-08-23 11:47:32,404] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-19T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-19T11:00:00+00:00
[2022-08-23 11:47:32,405] {logging_mixin.py:115} INFO - world
[2022-08-23 11:47:32,406] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:47:32,430] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211019T110000, start_date=20220823T114732, end_date=20220823T114732
[2022-08-23 11:47:32,484] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:47:32,530] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
